src/resources/users/user.interface.ts

```diff
@@ -0,0 +1,9 @@
+export interface IUser {
+  id: string;
+  name: string;
+  login: string;
+}
+
+export interface IUserDB extends IUser {
+  email: string;
+}
```

Comment:
At the moment we don't need to begin interfaces names with "I". And why do we need to have 2 interfaces here?

Reply:
Renamed as requested. Used both in user.router and user.model